content/en/logs/guide/azure-logging-guide.md

```diff
@@ -278,72 +278,33 @@ See [Diagnostic settings in Azure monitor][213] for more information.
+**Note**: Resources can only send logs to storage accounts in the same Azure region.
 
-1. If you haven't already set up [Azure Blob Storage][301], use one of the following methods to get started: 
-   - [Azure portal][302]
-   - [Azure Storage Explorer][303]
-   - [Azure CLI][304]
-   - [PowerShell][305]
-2. Set up the Datadog-Azure Function to forward logs from Blob Storage using the instructions below.
-3. Configure your Azure App Services to [forward their logs to Blob Storage][306].
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/CustomDeploymentBlade/uri/https%3A%2F%2Fddazurelfo.blob.core.windows.net%2Ftemplates%2Fforwarder.json)
 
-##### Create a function app
+After the deployment finishes, a storage account with [Azure Blob Storage][301] services will be deployed, along with a [Container App job][302] and [an environment][303], using the names and specifications chosen.
 
-If you already have a function app configured for this purpose, skip to [Add a new function to your Function App using the Event Hub trigger template](#add-a-new-function-to-your-function-app-using-the-azure-blob-storage-trigger-template).
 
-1. In the Azure portal, navigate to the [Function App overview][309] and click **Create**.
-2. In the **Instance Details** section, configure the following settings:  
-  a. Select the **Code** radio button  
-  b. For **Runtime stack**, select `Node.js`  
-  c. For **Version**, select `18 LTS`.  
-  d. For **Operating System**, select `Windows`.  
-3. Configure other settings as desired.
-4. Click **Review + create** to validate the resource. If validation is successful, click **Create**.
+##### Add diagnostic settings to send logs
+###### Resource logs
 
-##### Add a new function to your Function App using the Azure Blob Storage trigger template
+For each resource which you would like to [forward resource logs][304], when viewing the resource in the Azure portal, under **Monitoring**, select **Diagnostic settings**. Create a diagnostic setting, select all the categories you would like to forward, and under **Destination details**, select **Archive to a storage account**, and enter the details of the storage account created by the deployment.
 
-1. Select your new or existing function app from the [Function App overview][309].
-2. Under the **Functions** tab, click **Create**. 
-3. For the **Development environment** field, select **Develop in portal**.
-4. Under **Select a template**, choose [Azure Blob storage trigger][313].
-5. Select your **Storage account connection**.
-   **Note**: See [Configure a connection string for an Azure storage account][311] for more information.
-6. Click **Create**.
+Alternatively, you can browse all resources with diagnostic setting capability in the [Monitor | Diagnostic settings][305] page of the portal.
```

Comment:
```suggestion
Alternatively, you can view all resources that support diagnostic settings on the [**Monitor > Diagnostic settings**][305] page in the Azure portal.
```

Reply:
Doesn't have to be part of this PR, but we should also recommend [Policies](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings-policy)